.ops/performance-test/output.json

```diff
@@ -4,660 +4,668 @@
       "contains": "default",
       "type": "trend",
       "values": {
-        "avg": 488.27428080999994,
-        "max": 787.886004,
-        "med": 464.5849845,
-        "min": 323.809102,
-        "p(90)": 617.7130405,
-        "p(95)": 686.0770987499999
+        "avg": 2252.161498140624,
+        "max": 9104.424655,
+        "med": 1823.6385085000002,
+        "min": 97.474613,
+        "p(90)": 4267.477080300001,
+        "p(95)": 5131.440794249999
       }
     },
     "_api__item": {
       "contains": "default",
       "type": "trend",
       "values": {
-        "avg": -1000000000,
-        "max": -1000000000,
-        "med": -1000000000,
-        "min": -1000000000,
-        "p(90)": -1000000000,
-        "p(95)": -1000000000
+        "avg": 41.280713250000005,
+        "max": 57.599385,
+        "med": 41.7474725,
+        "min": 25.642178,
+        "p(90)": 50.0670646,
+        "p(95)": 53.47407705
       }
     },
     "_api_activities": {
       "contains": "default",
       "type": "trend",
       "values": {
-        "avg": 6557.076427409998,
-        "max": 10062.296899,
-        "med": 6023.648541500001,
-        "min": 5310.69271,
-        "p(90)": 8274.809629200005,
-        "p(95)": 9226.332316049999
+        "avg": 7550.652854734374,
+        "max": 18398.952399,
+        "med": 7487.960403,
+        "min": 1918.21193,
+        "p(90)": 11661.1986517,
+        "p(95)": 13263.264917749999
       }
     },
     "_api_activities_item": {
       "contains": "default",
       "type": "trend",
       "values": {
-        "avg": 906.1673087799999,
-        "max": 1577.457043,
-        "med": 821.3326,
-        "min": 623.407903,
-        "p(90)": 1261.5599443000006,
-        "p(95)": 1415.9576202
+        "avg": -15622746.995827295,
```

Comment:
🤨

Reply:
One of the failed requests is this:
![image](https://github.com/user-attachments/assets/2a392811-97ad-449a-bca2-323508361502)
![image](https://github.com/user-attachments/assets/857cf43e-0ac6-4c96-874b-ee60f034c7d1)